Core/include/Acts/Seeding/BinnedSPGroup.hpp

```diff
@@ -266,18 +266,15 @@ class BinnedSPGroup {
  public:
   BinnedSPGroup() = delete;
 
-  using GlobalPositionFunctor =
-      std::function<std::pair<Acts::Vector3, Acts::Vector2>(
-          const external_spacepoint_t&, float, float, float)>;
-
-  template <typename spacepoint_iterator_t>
+  template <typename spacepoint_iterator_t,
+	    typename callable_t>
   BinnedSPGroup<external_spacepoint_t>(
       spacepoint_iterator_t spBegin, spacepoint_iterator_t spEnd,
-      GlobalPositionFunctor toGlobal,
+      callable_t&& toGlobal,
```

Comment:
whats the reason to change this to an rvalue reference?

Reply:
It's a universal reference, so lvalue or rvalue according to what the end user passes